SU-SWS/stanford-jumpstart-deployer
# Update services-7.x-3.20 and smtp-7.x-1.17

# READY FOR REVIEW

# Summary
- Security updates for `services` and `smtp`

# Needed By (Date)
- Soon please

# Criticality
- How critical is this PR on a 1-10 scale? 9
- Affects all products. Before it's merged, sites will get one-offs of older, insecure versions. That makes me a sad panda.

# Steps to Test

1. Look at what Travis says
2. Check out this branch
3. `find stanford-jumpstart-deployer -name *.make | xargs grep -E "services\]\[version"`
4. `find stanford-jumpstart-deployer -name *.make | xargs grep -E "smtp\]\[version"`
5. Steps 3 and 4 will ensure that it's been caught everywhere

We've already tested the services functionality, so this is just to get it into the products.

# Affected Projects or Products
- Arrything

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/DEVOPS-97
- https://github.com/SU-SWS/Stanford-Drupal-Profile/pull/84
